rocketc/rocketc.py

```diff
@@ -0,0 +1,240 @@
+"""
+TO-DO: Write a description of what this XBlock is.
+"""
+import hashlib
+import pkg_resources
+import requests
+
+from xblock.core import XBlock
+from xblock.fields import Integer, Scope, String
+from xblock.fragment import Fragment
+from xblockutils.resources import ResourceLoader
+
+LOADER = ResourceLoader(__name__)
+
+@XBlock.wants("user") # pylint: disable=too-many-ancestors
+class RocketChatXBlock(XBlock):
+    """
+    TO-DO: document what your XBlock does.
+    """
+
+    # Fields are defined on the class.  You can access them in your code as
+    # self.<fieldname>.
+
+    count = Integer(
+        default=0, scope=Scope.user_state,
+        help="A simple counter, to show something happening",
+    )
+    rocket_chat_role = String(
+        default="user", scope=Scope.user_state,
+        help="The defined role in rocketChat"
+    )
+
+    url_prefix = "http://192.168.0.16:3000/api/v1"
```

Comment:
you can make this a global constant for now, and we should make it a configurable setting later on